### PR TITLE
[Api Management] Missing Examples for Operation and Refresh the older Ones

### PR DESCRIPTION
**Pull request by solankisamir**
- Fix 15 Errors for XmsExamplesRequired
- fix Issue https://github.com/Azure/azure-rest-api-specs/issues/1887
- Add example for Wsdl Import

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimbackends.json [hunk 1]
```diff
@@ -225,7 +230,7 @@
             "in": "body",
             "required": true,
             "schema": {
-              "$ref": "#/definitions/BackendUpdateParameters"
+              "$ref": "#/definitions/BackendUpdateContract"
```
Comment (dsgouda) [inline specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimbackends.json]: I'd recommend naming the properties model (`BackendUpdateParameters`) as something else and naming `BackendUpdateContract` as `BackendUpdateParameters` for better consistency

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimusers.json [hunk 2]
```diff
@@ -244,12 +249,6 @@
           "204": {
             "description": "The user details were successfully updated."
           },
-          "405": {
-            "description": "Administrator user cannot be modified.",
-            "schema": {
-              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
-            }
-          },
```
Comment (dsgouda) [inline specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimusers.json]: You can add this information in the description of default error response like [here](https://github.com/Azure/azure-rest-api-specs/blob/current/specification/mysql/resource-manager/Microsoft.DBforMySQL/2017-04-30-preview/mysql.json#L467)

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/examples/ApiManagementListProductGroups.json [hunk 3]
```diff
@@ -4,22 +4,44 @@
...
     "responses": {
         "200": {
             "body": {
                 "value": [
                     {
-                        "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.ApiManagement/service/apimService1/groups/administrators",
+                        "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.ApiManagement/service/apimService1/products/5600b57e7e8880006a060002/groups/5600b57e7e8880006a020001",
                         "type": "Microsoft.ApiManagement/service/products/groups",
-                        "name": "administrators",
+                        "name": "5600b57e7e8880006a020001",
```
Comment (dsgouda) [inline specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/examples/ApiManagementListProductGroups.json]: Just out of curiosity, this is just an example and not an actual name, correct?

**Review by dsgouda — CHANGES_REQUESTED** [review]
A few changes and questions

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimbackends.json (hunk 1 above)
Comment (solankisamir) [reply specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimbackends.json]: fixed

---
In reply to: [147817992](https://github.com/Azure/azure-rest-api-specs/pull/1945#discussion_r147817992) [](ancestors = 147817992)

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimusers.json (hunk 2 above)
Comment (solankisamir) [reply specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimusers.json]: fixed

---
In reply to: [147818888](https://github.com/Azure/azure-rest-api-specs/pull/1945#discussion_r147818888) [](ancestors = 147818888)

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/examples/ApiManagementListProductGroups.json (hunk 3 above)
Comment (solankisamir) [reply specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/examples/ApiManagementListProductGroups.json]: yes, customer can choose to provide a name or not.

If he does not, then we create for customer. The Name which is actually associated with the entity is "displayName" which can have spaces and is not restricted by naming conventions of URI pattern.

---
In reply to: [147819237](https://github.com/Azure/azure-rest-api-specs/pull/1945#discussion_r147819237) [](ancestors = 147819237)

Comment (dsgouda) [reply specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/examples/ApiManagementListProductGroups.json]: I understand. What concerns me is whether this is some sort of a service side secret. If it isn't, no issues

Comment (solankisamir) [reply specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/examples/ApiManagementListProductGroups.json]: No secret.

**Review by dsgouda — APPROVED** [review]
LGTM pending CI builds

**Pull request merged**